src/dmd/dtoh.d

```diff
@@ -2903,6 +2935,23 @@ const(char*) keywordClass(const Identifier ident)
         case "volatile":
             return "keyword in C++";
 
+        // Common macros
+        case "__STRICT_ANSI__":
+        case "__BASE_FILE__":
+        case "__FILE_NAME__":
+        case "__COUNTER__":
+        case "NULL":
```

Comment:
`NULL` by itself is not generally problematic, but if you `#include <stddef.h>`, then there'll be a conflict.  Though that goes for any pre-included header really.

Reply:
The generated header includes `stddef.h` by default to use `size_t`. The check is now restricted to `NULL` and `offsetof` (esp. the latter is not unlikely to appear in user code)